# get_angle_act_1.msg

int32[] finger_ids       
int32[] angles            
string[] finger_names     

